src/lib/datepicker/datepicker.ts

```diff
@@ -115,6 +115,10 @@ export class MatDatepickerContent<D> implements AfterContentInit {
   preserveWhitespaces: false,
 })
 export class MatDatepicker<D> implements OnDestroy {
+
+  /** An input indicating the type of the custom header component for the calendar, if set. */
+  @Input() customCalendarHeaderComponent: ComponentType<any>;
```

Comment:
`calendarHeaderComponent`

Reply:
done in 014777acc873ba3949a881bc1a002259c4665078